### ISSUE TITLE
Number of epochs and training time

### ISSUE THREAD

**Issue by kwea123** [ucbdrive/hd3#25]
I wonder if there is a reason why we need to train the model for so many epochs (1000s for KITTI and Sintel). Does the model still improve? Or can we shorten the number of epochs?

Currently I'm trying training on FlyingChairs, but due to the GPU memory problem I can only train at a speed of 1 hour/epoch. I know that if more GPUs are available (e.g. 8), this can be about ~5x faster, but still that requires huge amount of time if the number of epochs is more than 1000...

So to sum up, the question is
1. Is the large number of epochs necessary?
2. If it really is, how much did it take to train? (when you did the experiment)

Thank you!

**Comment by yzcjtr** [comment]
1. We followed some papers to determine the number of epochs. As I can recall, the result at 1000 epochs is somewhat better than 500 epochs.

2. We used 8 GPUs (Titan XP) to finetune the model on KITTI and it took roughly 1~2 days. The finetuning on Sintel is even faster, which took several hours.

**Comment by kwea123** [comment]
It's faster than I estimated. Thank you.

**Comment by yzcjtr** [comment]
You are welcome. Maybe you can try to shorten the training epochs on the synthetic datasets. The training schedule is flexible and we just figured out our own way in a limited time. But I think it might be redundant (in terms of efficiency) and suboptimal. Welcome to share your results if you find anything interesting.

**Comment by kwea123** [comment]
Do you have the validation set average EPE on datasets FlyingChairs and FlyingThings 3D (for flow)? These numbers are not in the paper. I can make some comparison on these datasets first.

**Comment by yzcjtr** [comment]
You can use our pretrained model on these datasets. The test script supports EPE calculation as well.
